engine/src/main/java/com/datatorrent/stram/util/LoggerUtil.java

```diff
@@ -355,4 +369,91 @@ private static boolean isErrorLevelEnable()
     return false;
   }
 
+  /**
+   * Adds Logger Appender
+   * @param name Appender name
+   * @param properties Appender properties
+   */
+  public static void addAppender(String name, Properties properties)
```

Comment:
Please change the method to return boolean to indicate success or failure to the caller.